testing/directives-test.js

```diff
@@ -1,15 +1,16 @@
-/*
 (function(){
    'use strict';
-    describe('Game square tests', function(){
+    describe('Directive tests', function(){
        var compile;
        var rootScope;
+       var gameSquare;
+       var squareNumber;
 
-        beforeEach(module('tombola.noughtsAndCrossesApp.directive'));
+        /*beforeEach(module('tombola.noughtsAndCrossesApp.directive'));*/
```

Comment:
Don't commit commented code, uncomment or remove. As my old boss used to say "Shit or get off the pot"